api/genomics/wdl/ImportGenomes.wdl

```diff
@@ -0,0 +1,276 @@
+version 1.0
+
+workflow ImportGenomes {
```

Comment:
Likewise, I believe these WDLs are copied - can you put a comment pointing to the original here as well?